app/helpers/account/tasks_helper.rb

```diff
@@ -1,2 +1,11 @@
 module Account::TasksHelper
+  def task_style(task)
```

Comment:
is it style or class?

Reply:
"style_for" is better?

Reply:
usage in code is like `class: task_style(task)`
method name should reflect what's it does/returns
in your case it'd be more like `class: task_class(task)` or something like that

Reply:
thanks